### ISSUE TITLE
Inconsistent Line Endings

### ISSUE THREAD

**Issue by SMAH1** [Dreamescaper/ServiceScan.SourceGenerator#14]
Windows 10 22H2
Visual Studio: Community 2202 17.12.2
dotnet project: 8.0
ServiceScan.SourceGenerator: 1.2.5

Hi
I config:
```csharp

    [GenerateServiceRegistrations(
    TypeNameFilter = "AAA.Singleton.*Service",
    AsImplementedInterfaces = true,
    Lifetime = ServiceLifetime.Singleton
    )]
    public static partial IServiceCollection AutomaticallyAddSingletonServices(this IServiceCollection services);
```
result is:

```csharp
using Microsoft.Extensions.DependencyInjection;

namespace AAA;

public static partial class AutomaticallyAddService
{
    public static partial IServiceCollection AutomaticallyAddSingletonServices(this IServiceCollection services)
    {
        return services
            .AddSingleton<AAA.Singleton.IUuidV7Service, AAA.Singleton.UuidV7Service>()
            .AddSingleton<AAASingleton.IDateTimeService, AAA.Singleton.DateTimeService>();
    }
}
```
and compile source, **everything is OK**.
but when open source file is generated, see warning:
![image](https://github.com/user-attachments/assets/0c3feff2-6b0c-41b3-84e9-5e079d1eb7f1)

this warning show when use '\n' instance of '\r\n'.
NOTE: Generated file must have two or more services added.

**Comment by Dreamescaper** [comment]
This issue is quite annoying. While it should be possible to generate consistant endings, VS shouldn't care at all about such things for generated files.
You can upvote this Roslyn bug:
https://github.com/dotnet/roslyn/issues/54633

**Comment by Dreamescaper** [comment]
Fixed in scope of #15 .
